hrui/js/hrui.js

```diff
@@ -12,7 +12,23 @@
       }
     });
   }
+
+  /**
+   * Update label 'for' attr to works with the datepicker
+   *
+   * @param  {jQuery Obejct} line [tr parent element]
+   */
+  function updateLabelFor(line) {
+    line.find('label').attr('for', line.find('.crm-form-date').attr('id'));
+  }
+
   $(document).on('crmLoad', function(e) {
+    if ($('.CRM_HRRecruitment_Form_HRVacancy').length === 1) {
+      var lines = $('.CRM_HRRecruitment_Form_HRVacancy tbody:first-child tr');
+      updateLabelFor($(lines.get(7)));
```

Comment:
I know this is all temporary, but targeting an element just based on the parent's position doesn't seem very reliable. Isn't there a better way to target?